platform/spire/src/virt.py

```diff
@@ -325,10 +338,10 @@ def generate_qemu_args(self):
         return args
 
     def check_module_loaded(self):
-        # TODO: don't blindly load kvm-intel; check type of system first
-        sudo("modprobe", "kvm", "kvm-intel")
+        # TODO: don't blindly load kvm_intel; check type of system first
+        modprobe("kvm", "kvm_intel")
 
-    def boot_with_io(self, phase, output_callback=None, text: bytes=None, delay=None):
+    def boot_with_io(self, phase, output_callback=None, text: bytes=None, delay=None, chardelay=None):
```

Comment:
This is getting really hacky.  Maybe we should try using [QMP](https://wiki.qemu.org/QMP) for this, which is designed for scripting.

Reply:
I suspect this may actually be a problem with the guest, not the host. But yeah, we should consider using a better tool at some point. I'm hoping to not have that hold up this PR, though.